modules/serverless-functions-secrets-configmaps-interactively-specialized.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-accessing-secrets-configmaps.adoc
+
+[id="serverless-functions-secrets-configmaps-interactively-specialized_{context}"]
+= Modifying function access to secrets and config maps interactively with specialized commands
```

Comment:
Should you remove the _interactively_ in this topic to better distinguish it from the _Modifying function access to secrets and config maps interactively_ section?

```
Modifying function access to secrets and config maps interactively
Modifying function access to secrets and config maps with specialized commands
```

Reply:
It's important that we keep "interactively", because we want to indicate that this is still the interactive way to modify function access, it's just we decrease the number of things we need to do once inside the interactive utility.